_ml-commons-plugin/api.md

```diff
@@ -143,18 +197,22 @@ The following example request registers a version `1.0.0` of an NLP sentence tra
 ```json
 POST /_plugins/_ml/models/_register
 {
-  "name": "all-MiniLM-L6-v2",
-  "version": "1.0.0",
-  "description": "test model",
-  "model_format": "TORCH_SCRIPT",
-  "model_config": {
-    "model_type": "bert",
-    "embedding_dimension": 384,
-    "framework_type": "sentence_transformers",
-  },
-  "url": "https://github.com/opensearch-project/ml-commons/raw/2.x/ml-algorithms/src/test/resources/org/opensearch/ml/engine/algorithms/text_embedding/all-MiniLM-L6-v2_torchscript_sentence-transformer.zip?raw=true"
+    "name": "all-MiniLM-L6-v2",
+    "version": "1.0.0",
+    "description": "test model",
+    "model_format": "TORCH_SCRIPT",
+    "model_group_id": "FTNlQ4gBYW0Qyy5ZoxfR",
+    "model_content_hash_value": "9376c2ebd7c83f99ec2526323786c348d2382e6d86576f750c89ea544d6bbb14",
```

Comment:
`model_content_hash_value` is mandatory, add this field to filed table above? Explain that the hash value is using sha-256